extension-manifest-v3/scripts/xcode-version.js

```diff
@@ -0,0 +1,33 @@
+// Open ./xcode/Ghostery.xcodeproj/project.pbxproj and find the line that looks like this:
```

Comment:
not sure if this still works but there are tools that are meant to do just this. Please check https://developer.apple.com/library/archive/qa/qa1827/_index.html